sudachipy/dictionarylib/doublearraylexicon.py

```diff
@@ -78,7 +78,7 @@ def size(self) -> int:
         return self.word_params.size
 
     def get_word_id(self, headword: str, pos_id: int, reading_form: str) -> int:
-        for wid in range(self.word_infos.size()):
+        for wid, _ in self.lookup(headword.encode('utf-8'), 0):
```

Comment:
oops.... I found that `lookup()` can't find split only words (連接ID=-1), since it won't be added to TRIE 

https://github.com/WorksApplications/SudachiPy/blob/7e5b501111920cef057b821412af595db248a7b8/sudachipy/dictionarylib/dictionarybuilder.py#L97